### ISSUE TITLE
Error with nested selectors starting with combinator

### ISSUE THREAD

**Issue by davidtheclark** [postcss/postcss-selector-parser#16]
Over at https://github.com/stylelint/stylelint/issues/348, @benfrain and @kungfusheep pointed out that the following syntax is breaking postcss-selector-parser:

```
@use postcss-nested;
.ml78-Anims_Target {
    > * { }
}
```

PostCSS doesn't trip up on it; and postcss-nested compiles it correctly; but this module throws the following error:

```
TypeError: Cannot read property '0' of undefined
    at Parser.combinator (C:\node_modules\stylelint\node_modules\postcss-selector-parser\dist\parser.js:144:42)
    at Parser.parse (C:\node_modules\stylelint\node_modules\postcss-selector-parser\dist\parser.js:375:26)
    at Parser.loop (C:\node_modules\stylelint\node_modules\postcss-selector-parser\dist\parser.js:342:22)
    at new Parser (C:\node_modules\stylelint\node_modules\postcss-selector-parser\dist\parser.js:87:21)
    at Processor.process (C:\node_modules\stylelint\node_modules\postcss-selector-parser\dist\processor.js:36:25)
    at C:\node_modules\stylelint\dist\rules\selector-no-id\index.js:41:10
    at C:\node_modules\postcss\lib\container.js:141:30
    at C:\node_modules\postcss\lib\container.js:102:26
    at Rule.each (C:\node_modules\postcss\lib\container.js:89:22)
    at Rule.eachInside (C:\node_modules\postcss\lib\container.js:101:21) 
```

Any chance the parser can support this kind of nested selector?

**Comment by ben-eb** [comment]
Yeah, the parser is a bit opinionated in this area, which makes sense looking at standard syntax, but not so much with nested syntax. :+1:

**Comment by ben-eb** [comment]
Released in `1.1.4`. Thanks for the report! :smile:

**Comment by davidtheclark** [comment]
Thanks for the quick response. I'll get that fix into stylelint this
evening.
